### ISSUE TITLE
ghc-stage1: could not execute: ghc-split

### ISSUE THREAD

**Issue by angerman** [snowleopard/shaking-up-ghc#81]
After fixing the `-I` of #79 with PR #80, I'm now stuck with:
```
ghc-stage1: could not execute: /Users/angerman/Projects/Haskell/ghc/inplace/lib/bin/ghc-split
shakeArgsWith   0.002s    0%                           
Function shake  0.028s    1%                           
Database read   0.195s    8%  ==                       
With database   0.010s    0%                           
Running rules   2.056s   89%  =========================
Total           2.292s  100%                           
Error when running Shake build system:
* utils/hpc/stage1/build/tmp/hpc-bin
* libraries/unix/stage1/build/HSunix-2.7.1.1.o
* libraries/unix/stage1/build/System/Posix/Terminal/Common.o
* libraries/base/stage1/build/Data/Bits.hi
* libraries/base/stage1/build/Data/Bits.o
* libraries/base/stage1/build/GHC/Enum.hi
* libraries/base/stage1/build/GHC/Enum.o
* libraries/integer-gmp/stage1/build/GHC/Integer.hi
* libraries/integer-gmp/stage1/build/GHC/Integer.o
* libraries/integer-gmp/stage1/build/GHC/Integer/Type.hi
* libraries/integer-gmp/stage1/build/GHC/Integer/Type.o
* libraries/ghc-prim/stage1/build/GHC/Types.hi
* libraries/ghc-prim/stage1/build/GHC/Types.o
user error (Development.Shake.cmd, system command failed
Command: inplace/bin/ghc-stage1 -hisuf hi -osuf o -hcsuf hc -static -hide-all-packages -no-user-package-db -this-package-key ghc-prim-0.5.0.0 -package-id rts -i -ilibraries/ghc-prim/stage1/build -ilibraries/ghc-prim/stage1/build/autogen -ilibraries/ghc-prim -Ilibraries/ghc-prim/stage1/build -Ilibraries/ghc-prim/stage1/build/autogen -I/Users/angerman/Projects/Haskell/ghc/rts/dist/build -I/Users/angerman/Projects/Haskell/ghc/includes -I/Users/angerman/Projects/Haskell/ghc/includes/dist-derivedconstants/header -optP-include -optPlibraries/ghc-prim/stage1/build/autogen/cabal_macros.h -this-package-key ghc-prim -XHaskell2010 -odir libraries/ghc-prim/stage1/build -hidir libraries/ghc-prim/stage1/build -stubdir libraries/ghc-prim/stage1/build -rtsopts -H32m -O2 -Wall -fwarn-tabs -split-objs -c libraries/ghc-prim/GHC/Types.hs -o libraries/ghc-prim/stage1/build/GHC/Types.o
Exit code: 1
Stderr:
ghc-stage1: could not execute: /Users/angerman/Projects/Haskell/ghc/inplace/lib/bin/ghc-split
)
```

**Comment by angerman** [comment]
Ahh well, we need to fix `ghc-split` then. The current source implies it's not implemented:
```
shake-build/src/GHC.hs
92:-- TODO: The following utils are not implemented yet: unlit, driver/ghc-split

shake-build/src/Rules/Generators/ConfigHs.hs
9:-- TODO: do we need ghc-split? Always or is it platform specific?
```

**Comment by snowleopard** [comment]
See https://github.com/snowleopard/shaking-up-ghc/issues/83#issuecomment-168501954 for a temporary workaround.

**Comment by angerman** [comment]
This would be fixed with #138

**Comment by snowleopard** [comment]
I tested this on Windows. Also Travis Linux is happy about it.
